libctru/source/services/soc/soc_gethostbyaddr.c

```diff
@@ -0,0 +1,72 @@
+#include "soc_common.h"
+#include <netdb.h>
+
+#define MAX_HOSTENT_RESULTS 16
+static struct hostent SOC_hostent;
+static char           *SOC_hostent_results[MAX_HOSTENT_RESULTS+1];
+static char           *SOC_hostent_alias = NULL;
+
+int h_errno = 0;
```

Comment:
`h_errno` is already defined in [soc_gethostbyname](https://github.com/Megazig/ctrulib/blob/master/libctru/source/services/soc/soc_gethostbyname.c#L9). Pinging @mtheall to see how can this be fixed.